hack/update-codegen.sh

```diff
@@ -277,19 +286,15 @@ gen-conversions() {
       --extra-dirs github.com/jetstack/cert-manager/pkg/internal/apis/meta/v1
     
     # Workaround (see: https://github.com/kubernetes/kubernetes/issues/101567)
-    sed_args=("${global_sed_args[@]}")
-    sed_args+=('s/\+build \!ignore_autogenerated/TEMP_REPLACE_IGNORE_AUTOGENERATED/g' "pkg/internal/apis/acme/$v/zz_generated.conversion.go")
-    sed "${sed_args[@]}"
+    remove_go_autogen_tags "pkg/internal/apis/acme/$v/zz_generated.conversion.go"
 
     "$conversiongen" --go-header-file hack/boilerplate/boilerplate.generatego.txt \
       --input-dirs "github.com/jetstack/cert-manager/pkg/internal/apis/certmanager/$v" \
       -O zz_generated.conversion \
       --extra-dirs "github.com/jetstack/cert-manager/pkg/internal/apis/meta/v1,github.com/jetstack/cert-manager/pkg/internal/apis/acme/$v"
 
     # Workaround (see: https://github.com/kubernetes/kubernetes/issues/101567)
-    sed_args=("${global_sed_args[@]}")
-    sed_args+=('s/\+build \!ignore_autogenerated/TEMP_REPLACE_IGNORE_AUTOGENERATED/g' "pkg/internal/apis/certmanager/$v/zz_generated.conversion.go")
-    sed "${sed_args[@]}"
+    remove_go_autogen_tags "pkg/internal/apis/certmanager/$v/zz_generated.conversion.go"
```

Comment:
Nice that you've pulled this repetitive code into a bash function. Much easier to read.